RecipeMatch/Models/GridRecipeCell.h

```diff
@@ -0,0 +1,17 @@
+//
+//  GridRecipeCell.h
+//  RecipeMatch
+//
+//  Created by ellabuysse on 7/11/22.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface GridRecipeCell : UICollectionViewCell
```

Comment:
collection view cells are actually UIViews, so it actually makes more sense for them to live under the Views directory you have :)